packages/data-contracts/src/lib.rs

```diff
@@ -19,6 +20,7 @@ pub enum SystemDataContract {
     FeatureFlags = 2,
     DPNS = 3,
     Dashpay = 4,
+    Wallet = 5,
```

Comment:
Wallet is too generic of a name

Reply:
@HashEngineering any idea?

Reply:
We could name it "Transaction" instead since the data contract only applies to transactions at this time.

At this point, we don't know if other "Wallet" level documents could or would be added to this data contract later.

Reply:
Maybe `WalletUtilities`?

Reply:
WalletUtils
WalletUtilitites

Reply:
Renamed